examples/llm.c/export.py

```diff
@@ -62,7 +63,7 @@
     if v.lazydata.base.buffer not in used_buffers: print(f"UNUSED: {k}")
     if v.grad is not None: grad_state_dict['grad_'+k] = v.grad
   state_dict.update(grad_state_dict)
-  state_dict.update({'adam_b1': optimizer.b1, 'adam_b2': optimizer.b2, 'adam_t': optimizer.t, 'adam_lr': optimizer.lr})
+  state_dict.update({'adam_b1': optimizer.b1_t, 'adam_b2': optimizer.b2_t, 'adam_lr': optimizer.lr})
```

Comment:
are you sure you want to update `adam_b1` with `b1_t`? `bt_t` tracks `b1 ** t`

Reply:
b1 and b2 are float literals now - They don't have an independent schedule

Reply:
i see, update the key to `adam_b1_t` and `adam_b2_t` to match?